data/kz/4.0.dict

```diff
@@ -6,67 +6,67 @@
 % The empty word.
 EMPTY-WORD.zzz: ZZZ-;
 
-мен.pron: {S1s+};
-сен.pron: {S2s+};
-сіз.pron: {Sr2s+};
-ол.pron: {S3s+};
-
-біз.pron: {S1p+};
-сендер.pron: {S2p+};
-сіздер.pron: {Sr2p+};
-олар.pron: {S3p+};
-
-менің.pron: {OF1+};
-сенің.pron: {OF2+};
-сіздің.pron: {OFr2+};
-оның.pron: {OF3+};
-
-біздің.pron: {OF1p+};
-сендердің.pron: {OF2p+};
-сіздердің.pron: {OFr2p+};
-олардың.pron: {OF3p+};
-
+мен.pron: S1s+;
+сен.pron: S2s+;
+сіз.pron: Sr2s+;
+ол.pron: S3s+;
+
+біз.pron: S1p+;
+сендер.pron: S2p+;
+сіздер.pron: Sr2p+;
+олар.pron: S3p+;
+
+менің.pron: OF1+;
+сенің.pron: OF2+;
+сіздің.pron: OFr2+;
+оның.pron: OF3+;
+
+біздің.pron: OF1p+;
+сендердің.pron: OF2p+;
+сіздердің.pron: OFr2p+;
+олардың.pron: OF3p+;
 
 %adjectives beautiful white red
 әдемі.= ақ.= қызыл.=: {AO+} & {EI-};
 
-
 %nouns human girl boy dog cat
```

Comment:
There is no requirement that the documentation, notes or comments be in English. They can be in any language at all. Like swahili, for example.